### PR TITLE
suites/rbd: added replication stress test workunit

### PR DESCRIPTION
**Pull request by dillaman**
Signed-off-by: Jason Dillaman <dillaman@redhat.com>

### CODE REVIEW

**Comment by dillaman** [comment]
http://pulpito.ceph.com/jdillaman-2016-05-19_16:51:12-rbd-master---basic-mira/202478/

**Comment by dillaman** [comment]
@jdurgin when you get a chance

File: suites/rbd/mirror/workloads/rbd-mirror-stress-workunit.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+meta:
+- desc: run the rbd_mirror.sh workunit to test the rbd-mirror daemon
```
Comment (jdurgin) [inline suites/rbd/mirror/workloads/rbd-mirror-stress-workunit.yaml]: description doesn't mention stress

**Comment by dillaman** [comment]
fixed

**Pull request merged**